### DISCUSSION TITLE
How to remove PDF/A-standard from a PDF-file?

### DISCUSSION THREAD

**Discussion by roboterwithsenf** [qpdf/qpdf#847] [Q&A]
I have a pdf-file which is saved in the PDF/A-standard. I know how to work and ignore this standard with Adobe Acrobat, but I'd rather would like to remove the PDF/A-standard completely from the file and save it as an "normal" PDF-document. Is there any possibility with qpdf to do this?

**Comment by jbarlow83** [comment]
Can't be done from command line QPDF.

Easier method: QPDF you could drop the document catalog's /Metadata and /OutputIntents dictionary from qpdf.getRoot().removeKey(). If you don't care about metadata or print settings, you can probably remove those.

To do it more surgically you need to remove the XML namespace "http://www.aiim.org/pdfa/ns/id/" from the XMP metadata, and any output intents with "/S /GTS_PDFA*". pikepdf (which uses libqpdf) remove the XMP metadata.
